code/index.html

```diff
@@ -1,21 +1,55 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="favicon.ico" type="image/x-icon"/>
+    <title>Kreis4 Flower Shop - Order Flowers Online</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
 
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
-  </head>
-  <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+    <menu>
+        <a href="./index.html">Home</a>
+        <a href="./our-flowers.html">Flowers</a>
+        <a href="./subscription.html">Subscription</a>
+        <a href="./faqs.html">FAQs</a>
+        <a href="./about.html">About Us</a>
+    </menu>
 
-      <img src="example-recipe-background.png" alt="Banana Bread">
+    <header>
+    <h1>Welcome to Kreis4 Flowers<br>Fresh Flowers All Year Round</h1>
+    </header>
 
-    -->
-    <h1>Hello there! 👋🏼</h1>
-  </body>
-</html>
\ No newline at end of file
+    <div class="main-block">
+        <h2>Select your Favorite Regional Flowers</h2>
+        <p>Either for pick-up or home-delivery, select from a wide variety of the most beautiful flowers, hand-picked from near Zurich.</p>
+        <p>One-time order or subscription model, we got you covered!</p>
+    </div>
+
+    <div class="highlighter-block"><h2>Our Subscription Model</h2>
+        <p>Select between the following models: <br>
+            <ul>
+                <li>Weekly Delivery</li>
+                <li>Bi-Weekly Delivery</li>
+                <li>Monthly Delivery</li>
+            </ul>
+        </p>
+        <p class="note">Of course, you can pause your subscription anytime.</p>
+    </div>
+
+<div class="highlighter-block">
+    <h2>Most-Ordered Flowers</h2>
+    <p>These flowers have been ordered most this season:
+        <ol>
+            <li><a href="https://en.wikipedia.org/wiki/Peony">Peony</a></li>
+            <li>Lily</li>
+            <li>Freesia</li>
+        </ol>
+    </p>
+    <p class="note">Check out the links to learn more</p>
+</div>
+
+<iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/hyXRyCbaiNk" frameborder="0"></iframe>
```

Comment:
Nice with a video! Just try to keep the styling to the CSS-file to be able to access it more quickly